packages/components/src/ErrorPage/ErrorPage.tsx

```diff
@@ -0,0 +1,98 @@
+import React, { HTMLAttributes } from "react"
+import { FormattedMessage, useIntl } from "@cultureamp/i18n-react-intl"
+import classNames from "classnames"
+import { BrandMoment } from "@kaizen/brand-moment"
+import { BrandMomentError } from "@kaizen/draft-illustration"
+import { Paragraph } from "@kaizen/typography"
+import { ArrowRightIcon } from "~components/SVG/icons/ArrowRightIcon"
+import { EmailIcon } from "~components/SVG/icons/EmailIcon"
+import { OverrideClassName } from "~types/OverrideClassName"
+import { useErrorMessages } from "./hooks"
+import styles from "./ErrorPage.module.scss"
+
+const getMailToHref = (code: number): string => {
+  const supportEmail = "support@cultureamp.com"
+  const subject = "Houston we have a problem"
+  const body = `Hi there,\n\nI received a ${code} error page while I was trying to...`
+  return encodeURI(`mailto:${supportEmail}?subject=${subject}&amp;body=${body}`)
+}
+
+const HOME_HREF = "/app/home"
+
+export type ErrorPageProps = {
+  code: number
+  title?: string
+  message?: React.ReactNode | string
+  callToAction?: {
+    onContactSupport: () => void
+    homeHref?: string
+  }
```

Comment:
Normally I would steer away from defining props in this manner and instead allow the consumer to pass through a render function.  But we do want this component to have an opinion on how our error pages should look and what they should do, so being able to change just these things may be okay.

It may also not be clear that the call to action will always appear even if you don't set anything, but again, maybe that's not a biggie. Its intention is to be an override.

Reply:
Yeah, a bit confusing, but for now should be okay.